rust/stackablectl/src/cmds/operator.rs

```diff
@@ -159,6 +184,9 @@ pub enum CmdError {
         source: namespace::Error,
         namespace: String,
     },
+
+    #[snafu(display("OCI error"))]
+    OciError { source: oci::Error },
```

Comment:
note: I feel like this should be part of `helm::Error`. Internally, this can still be an OCI-specific error enum.

Reply:
Tried. Failed :(

Reply:
As discussed, we'll address this here: https://github.com/stackabletech/stackable-cockpit/issues/347